node/test/runtime/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "polkadot-runtime-test"
+version = "0.9.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+
+test-runner = { git = "https://github.com/paritytech/substrate", branch = "master" }
+
+system = { package = "frame-system", git = "https://github.com/paritytech/substrate", branch = "master" }
+support = { package = "frame-support", git = "https://github.com/paritytech/substrate", branch = "master" }
+benchmarking = { package = "frame-benchmarking", git = "https://github.com/paritytech/substrate", branch = "master" }
+
+transaction-payment = { package = "pallet-transaction-payment", git = "https://github.com/paritytech/substrate", branch = "master" }
+collective = { package = "pallet-collective", git = "https://github.com/paritytech/substrate", branch = "master" }
+democracy = { package = "pallet-democracy", git = "https://github.com/paritytech/substrate", branch = "master" }
+
+polkadot-primitives = { version = "0.9", path = "../../../primitives" }
```

Comment:
Is there a reason we tie this to a version?

Reply:
not really, no